builder-base/install.sh

```diff
@@ -178,7 +178,8 @@ setupgo() {
 
 setupgo "${GOLANG113_VERSION:-1.13.15}"
 setupgo "${GOLANG114_VERSION:-1.14.15}"
-setupgo "${GOLANG115_VERSION:-1.15.10}"
+setupgo "${GOLANG115_VERSION:-1.15.11}"
+setupgo "${GOLANG115_VERSION:-1.16.3}"
```

Comment:
I think this should be 
`${GOLANG116_VERSION:-1.16.3}` ?

Reply:
Good catch!